include/ginkgo/config.hpp.in

```diff
@@ -99,4 +99,10 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 // clang-format on
 
 
+/* Do we need to use blocking communication in our SpMV? */
```

Comment:
only for spmv?

Reply:
That is the only part which caused trouble. I'm not sure if that is something general about `MPI_Ialltoallv`. Perhaps it makes sense to handle `communicator::i_all_to_all_v` in the same way.